easybuild/tools/systemtools.py

```diff
@@ -873,6 +881,37 @@ class LINKMAP(ctypes.Structure):
     return libpath.decode('utf-8')
 
 
+def find_library_path(lib_filename):
+    """
+    Search library by file name in the system
+    Return absolute path to existing libraries
+
+    :params lib_filename: name of library file
+    """
+
+    lib_abspath = None
+    os_type = get_os_type()
+
+    try:
+        lib_obj = ctypes.cdll.LoadLibrary(lib_filename)
+    except OSError:
+        _log.info("Library '%s' not found in host system", lib_filename)
+    else:
+        # ctypes.util.find_library only accepts unversioned library names
+        if os_type == LINUX:
+            # find path to library with dlinfo
+            lib_abspath = locate_solib(lib_obj)
+        elif os_type == DARWIN:
+            # ctypes.macholib.dyld.dyld_find accepts file names and returns full path
+            lib_abspath = ctypes.macholib.dyld.dyld_find(lib_filename)
+        else:
+            raise EasyBuildError("Unknown host OS type: %s", os_type)
+
+        _log.info("Found absolute path to %s: %s" % (lib_filename, lib_abspath))
```

Comment:
Please make this log statement "lazy" (only template when the log message is actually triggered)

```suggestion
      _log.info("Found absolute path to %s: %s", lib_filename, lib_abspath)
```

Reply:
Looks like you unintentionally unindented this suggestion?

Reply:
I edited the suggestion by @boegel, fixed :+1: